tests/torchtune/utils/test_distributed.py

```diff
@@ -194,7 +196,7 @@ def test_lora_fsdp_wrap(self):
 
         adapter_params = get_adapter_params(model)
         set_trainable_params(model, adapter_params)
-        num_lora_ab, num_transformer_layers = _get_n_lora_and_tformer_layers(model)
+        num_lora, num_transformer_layers = _get_n_lora_and_tformer_layers(model)
```

Comment:
I saw multiple changes to test_lora_fsdp_wrap test but the unit test doesn't pass on those changes.

Reply:
Sorry can you elaborate? Do you mean changes when you merge with latest main? Or changes from Calvin's initial version?

Reply:
There are several changes on test_distributed.py compared with master and I think it's either done by Calvin or your FSDP refactor work. Those tests are broken and I don't have context on why we need these changes

Reply:
OK yeah I can take a look. The only changes I made were [these ones](https://github.com/pytorch/torchtune/pull/1115/commits/dd694cef19a340a0b181778ae296ee2da5336cc4#diff-a4e7291c0c5acab6a14537dd1c67bf0406396115dafb7459149fe08fd6502758)

Reply:
I fixed the test_distributed.py issue by reverting some of Calvin's change on _distributed.py, which are unnecessary after the DoRA refactor